packages/client-node/src/utils/stream.ts

```diff
@@ -1,5 +1,8 @@
 import Stream from 'stream'
 
+// See https://github.com/v8/v8/commit/ea56bf5513d0cbd2a35a9035c5c2996272b8b728
+const MaxStringLength = Math.pow(2, 29) - 24
```

Comment:
I'm just curious: is it relevant for all supported nodejs versions?

Reply:
and also browsers, as it comes from the V8 itself